FirebaseAppCheck/Interop/FIRAppCheckInterop.h

```diff
@@ -31,6 +31,10 @@ NS_SWIFT_NAME(AppCheckInterop) @protocol FIRAppCheckInterop
                     completion:(FIRAppCheckTokenHandlerInterop)handler
     NS_SWIFT_NAME(getToken(forcingRefresh:completion:));
 
+/// Retrieve a new limitedUseToken
```

Comment:
super nit: Maybe something like "Retrieve a new limited-use Firebase App Check token." to match up with the documentation for `getTokenForcingRefresh:completion:`?

I don't feel strongly though since `FIRAppCheckInterop` is only used internally and this already gets the point across.

Reply:
Ya sure I can complete the comment to give more context